scripts/eval/eval.py

```diff
@@ -297,10 +299,14 @@ def main(cfg: DictConfig) -> Tuple[List[Trainer], pd.DataFrame]:
                                              'loggers',
                                              must_exist=False,
                                              default_value={})
-    icl_subset_num_batches: int = pop_config(cfg,
-                                             'icl_subset_num_batches',
-                                             must_exist=False,
-                                             default_value=None)
+    subset_num_batches: int = pop_config(cfg,
+                                         'subset_num_batches',
```

Comment:
Any particular reason to not just keep the naming consistent with train and call this `eval_subset_num_batches`?